lib/formatters.rb

```diff
@@ -204,7 +219,17 @@ def footer
     def convert(data, _first)
       CSV.generate do |csv|
         data.each do |o|
-          csv << o.values.map { |v| v == '' ? nil : v }
+          row = []
+          o.each do |k, v|
+            if k == "ec" || k == "go"
+              v.first.keys.each do |key|
+                row << (v.map { |el| el[key] }).join(" ")
+              end
+            else
+              row << (v == '' || k == "ec" || k == "go" ? nil : v)
```

Comment:
Why the ec and go check here? Aren't those handled by the initial `if` check?

_[Original comment](https://github.ugent.be/unipept/unipept-cli/pull/55#discussion_r21551) by @bmesuere on Thu May 23 2019 at 10:53._